mlflow/server/js/src/experiment-tracking/components/ExperimentPage.js

```diff
@@ -240,11 +245,12 @@ export class ExperimentPage extends Component {
   /*
     If this function returns true, the ExperimentView should nest children underneath their parents
     and fetch all root level parents of visible runs. If this function returns false, the views will
-    not nest children or fetch any additional parents.
+    not nest children or fetch any additional parents. Will always return true if the orderByKey is
+    'attributes.start_time'
   */
   shouldNestChildrenAndFetchParents() {
     const { orderByKey, searchInput } = this.state.persistedState;
-    return !orderByKey && !searchInput;
+    return (!orderByKey && !searchInput) || orderByKey === DEFAULT_ORDER_BY_KEY;
```

Comment:
Should we directly use [`AttributeColumnSortKey.DATE`](https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/experiment-tracking/components/ExperimentViewUtil.js#L212) instead of `DEFAULT_ORDER_BY_KEY`? Since the default order_by_key might change but we would still want the nested runs to be fetched if the orderByKey is startTime.

Reply:
Added the test case and used the SortKey.DATE key directly